src/atoms/Input/type.d.ts

```diff
@@ -0,0 +1,65 @@
+export interface InputWrapperProps {
+  children: ReactNode;
+  inputWrapperStyle?: React.CSSProperties;
+  disabled?: boolean;
+  ref?: React.Ref<HTMLDivElement>;
+}
+
+export type Variant = "halfWidth" | "fullWidth" | "default" | "line";
+export type State = "error" | "success" | "untouched";
```

Comment:
move commonly used variables to common folder please.
implies on all variables

Reply:
ok after common folder is build